_sass/jekyll-theme-midnight.scss

```diff
@@ -160,6 +160,13 @@ pre {
     margin: 0;
     padding: 0;
   }
+
+  u {
```

Comment:
In the code above you're using a `<span>`, not a `<u>`. Surely you meant the former, right?

Also, note that the PR description also says `<u>` instead of `<span>`.

Reply:
Thanks, I missed this. It should be `'<u>' + c + '</u>'`.

Also I just noticed that I used a tempalte string. I've gotten too used to TypeScript :D